---
layout: default
---

<section class="jumbotron research">
    <div class="container">
        <div class="row text-center">
            <h2>research</h2>
        </div>
    </div>
</section>
<section class="container">
    <div class="row">
        <p>In my research I use genome-wide approaches to address broadly relevant questions in the evolution of birds and mammals:</p>

        <h3>the impact of enhancer variants</h3>
        <p>To study the effects of genetic variation on enhancer activity, I analysed human-specific substitutions that reside in putative enhancers for their regulatory activity (<a href="https://doi.org/10.1101/865519">Uebbing, Gockley <i>et al.</i> 2019</a>). The tested enhancers fall into two categories: Human Accelerated Regions (HARs), which are conserved, non-coding regions that show an increased number of sequence substitutions in human, and Human Gained Enhancers (HGEs), which show increased histone marks for enhancer activity in humans compared to rhesus macaque and mouse. To test the substitutions for their regulatory effect, my co-author Jake Gockley and I used <a href="https://doi.org/10.1038/nbt.2137">Massively Parallel Reporter Assays</a> comparing human and chimpanzee enhancer alleles to first identify active and differentially active enhancers and then isolate the specific, regulatory effects of single human-specific substitutions. This way I identified over 400 substitutions with quantifiable effects on regulatory activity in cortex development that are distributed across the genome (<a href="https://doi.org/10.1101/865519">Uebbing, Gockley <i>et al.</i> 2019</a>). The next challenge is to connect these substitutions to their target genes and study their biological effects.</p>

        <h3>gene expression on avian sex chromosomes</h3>
        <p>In a pair of dimorphic sex chromosomes, one of the chromosomes has decayed during the course of evolution and lost much of its genetic information. This is problematic because the heterogametic sex will suffer from having half the number of sex chromosomal genes and their products compared to the other sex, and compared to autosomes. In many organisms, dosage compensation systems have been found that re-establish dosage equality. Birds, however, lack such a mechanism so that female birds have less gene products from their sex chromosomes. I investigated this by analysing genome-wide transcriptome and proteome data from different bird species. This confirmed that birds do not compensate gene dosage on mRNA level (<a href="http://dx.doi.org/10.1093/gbe/evt114">Uebbing <i>et al.</i> 2013</a>) and showed that this neither is the case on protein level (<a href="http://dx.doi.org/10.1093/molbev/msv147">Uebbing, Konzer, Xu <i>et al.</i> 2015</a>). I also used gene expression data to annotate the avian W chromosome (<a href="http://dx.doi.org/10.1038/ncomms6448">Smeds <i>et al.</i> 2014</a>) and to identify and analyse their pseudo-autosomal region (<a href="http://dx.doi.org/10.1038/ncomms8330">Smeds <i>et al.</i> 2015</a>).</p>

        <h3>gene expression diversity and divergence</h3>
        <p>In this project I studied of the origin and divergence of regulatory diversity in natural bird populations. I compared population samples of the transcriptomes of two <i>Ficedula</i> flycatcher species which speciated less than one million years ago. I found very little expression divergence in this young species pair and no support for predicted correlations of expression divergence with sequence divergence or pleiotropy (<a href="http://dx.doi.org/10.1111/mec.13596">Uebbing <i>et al.</i> 2016</a>). Data from this effort was also used in a large study on genomic divergence in these flycatchers (<a href="http://dx.doi.org/10.1038/nature11584">Ellegren <i>et al.</i> 2012</a>). I further used RNA-seq data to quantify allele-specific gene expression in natural flycatcher populations. This showed that there is a substantial amount of expression diversity within wild populations that may provide a starting point for gene expression divergence (<a href="https://doi.org/10.1093/gbe/evx080">Wang <i>et al.</i> 2017</a>).</p>
    </div>
</section>